tableau-databricks/manifest.xml

```diff
@@ -56,8 +56,8 @@ limitations under the License.
         <customization name='CAP_ODBC_FETCH_CONTINUE_ON_ERROR' value='no' />
         <customization name='CAP_ODBC_FORCE_SINGLE_ROW_BINDING' value='no' />
         <customization name='CAP_ODBC_METADATA_STRING_LENGTH_UNKNOWN' value='no' />
-        <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLFOREIGNKEYS_API' value='yes' />
-        <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLPRIMARYKEYS_API' value='yes' />
+        <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLFOREIGNKEYS_API' value='no' />
+        <customization name='CAP_ODBC_METADATA_SUPPRESS_SQLPRIMARYKEYS_API' value='no' />
```

Comment:
It may be good to document the sequence of metadata operations before and after the change.